app/services/authorization_service.rb

```diff
@@ -59,8 +59,7 @@ def build_user(identity)
       )
       user.name = auth.info.nickname if user.name.blank?
       user.skip_confirmation!
-      user.remember_me!
-      user.remember_me = true
```

Comment:
Setting `remember_me` doesn't change anything, because `User#remember_me` always returns `true`
https://github.com/thepracticaldev/dev.to/blob/dbac5f6d190913a3dab669d94d6ba73dd8e09a94/app/models/user.rb#L273-L275